### DISCUSSION TITLE
Ender 3V2 / Sprite Pro / CR-Touch - Repetier Server

### DISCUSSION THREAD

**Discussion by kadi1991** [mriscoc/Ender3V2S1#713] [General]
hey Guys,

i was looking forward to use the newest mriscoc firmware for my ender 3v2 with sprite pro extruder and cr-touch. the firmware is amazing with all the options and menus! respect for this work an thank you very much!

i got now a small problem. ill use my ender with the repetier server software for remote control an i dont want to switch to another software. but the repetier server cannot detect my ender. is there a special mod in work, to fix this problem?

thank you so much!

**Comment by mriscoc** [comment]
This firmware set the Serial USB speed to 250000 baud. Configure your software to use that speed.
